### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/intro/install.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#814] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/intro/install.html

要求 环境: LTS 版本 的 Node.js（18、20） 包管理器: npm >= 8、yarn >= 2 或 pnpm >= 7 使用 Vue3 的 VuePress2 项目 安装 在 [dir] 文件夹内新建 vuepress-theme-hope 项目: 要将 vuepress-theme-hope 作为文档构建器添加到现有项目中，请在项目根...

https://theme-hope.vuejs.press/zh/guide/intro/install.html

**Comment by wangwangarmy** [comment]
你好，每个页面顶上的索引（此页内容），可以更改样式吗，比如改一下字体颜色等等。想问一下这个对应的代码在哪里呀。

**Reply by wangwangarmy** [reply]
或者想把这个此页内容放在页面的右侧啥的

**Comment by songlongkuan** [comment]
Copyright © 2025 Mr.Hope  这个在哪去掉啊？
